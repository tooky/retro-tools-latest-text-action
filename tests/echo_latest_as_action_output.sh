cp -R ../../../doc .
run-action $(pwd)
