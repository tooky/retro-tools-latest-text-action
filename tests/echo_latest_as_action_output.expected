cp -R ../../../doc .
run-action $(pwd)
pwd
::set-output name=text::# 2020-08-05 ## Who - @tooky  ## What happened  - Emerged from nothing  ## Insights  ## Actions
